6dControl.hpp

```diff
@@ -27,8 +28,25 @@ namespace auv_control{
     struct LinearAngular6DPIDState{
         PIDState linear[3];
         PIDState angular[3];
+        base::Time time;
     };
-}    
+
+    struct LinearAngular6DPID{
+        motor_controller::PID linear[3];
+        motor_controller::PID angular[3];
+    };
+
+    struct LinearAngular6DCommandStatus
```

Comment:
This header is parsed / generated by orogen, stick to public interface types. Put private structs somewhere else.